src/test/java/BaseTest.java

```diff
@@ -29,21 +29,24 @@ public void setUpBrowser(String baseUrl){
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         driver.manage().window().maximize();
         url=baseUrl;
-        driver.get("baseUrl");
+        driver.get("url");
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224927152-f3608fb6-09c5-47e6-86d5-99d82917b50c.png)

**HINT**

![Test status](https://user-images.githubusercontent.com/104761482/224927263-96996334-2d33-443f-bd83-0d58a0e8b413.png)

![Test error_](https://user-images.githubusercontent.com/104761482/224927304-c110f878-fefd-45a6-8024-6d98f6dac7a7.png)

**driver.get(url);**

![image](https://user-images.githubusercontent.com/104761482/224927578-adb30e21-a500-47f4-955a-889a164925de.png)